### PR TITLE
Added handling of config file (~/.ralph-cli/config.toml).

### PR DESCRIPTION
**Pull request by xor-xor**

### CODE REVIEW

File: actions.go [hunk 1]
```diff
@@ -2,54 +2,44 @@ package main
...
 )
 
 // PerformScan runs a scan of a given host using a set of scripts.
 // At this moment, we assume that there will be only one script here (idrac.py),
 // and that only MAC addresses will be created/updated/deleted in Ralph.
-func PerformScan(addrStr string, scripts []string, dryRun bool, cfgDir string) {
+func PerformScan(addrStr string, scripts []string, dryRun bool, cfg *Config, cfgDir string) {
 	if dryRun {
-		fmt.Println("Running in dry-run mode, no changes will be saved in Ralph.")
+		// TODO(xor-xor): Wire up logger here.
```
Comment (mkurek) [inline actions.go]: what about this TODO? :)

File: config.go [hunk 2]
```diff
@@ -34,56 +55,135 @@ func GetCfgDirLocation(baseDir string) (string, error) {
...
+func GetConfig(cfgFile string) (*Config, error) {
+	cfg, err := readConfig(cfgFile)
+	if err != nil {
+		return nil, err
+	}
+	err = cfg.validate()
+	if err != nil {
+		return nil, err
+	}
+	cfg.getDefaults()
```
Comment (mkurek) [inline config.go]: what about environment variables? could we support them? For example we'll read config from toml file, then apply defaults, and then apply env variables on top of that (they will have priority over config file). Then we could specify env variables in during ralph-cli calling, ex. `MANAGEMENT_USER_NAME=admin MANAGEMENT_USER_PASSWORD=secret ralph-cli scan ...` @xor-xor what do you think about it?

File: actions.go (hunk 1 above)
Comment (xor-xor) [inline actions.go]: Not everything at once - there's a separate task for logger etc. (see RAS-652 in our internal backlog).

File: config.go (hunk 2 above)
Comment (xor-xor) [inline config.go]: See RAS-697 ;)

**Comment by quamilek** [comment]
🆗

**Comment by mkurek** [comment]
:ok:

**Pull request merged**